config/initializers/zeitwerk.rb

```diff
@@ -0,0 +1,15 @@
+if Rails.autoloaders.main
+  Zeitwerk::Loader.default_logger = method(:puts)
+  Rails.autoloaders.main.logger = Logger.new(STDOUT)
+
+  # These specific directories are for code organization, not namespacing:
+  Rails.autoloaders.main.collapse(Rails.root.join("lib/manageiq/reporting/charting"))
+  Rails.autoloaders.main.collapse(Rails.root.join("lib/ansible/runner/credential"))
```

Comment:
I presume we should fix these in a follow up to be properly namespaced?

Reply:
yeah, probably...  I have a few things in here that will likely need followup issues.